finals-2024-01-15/code/5100-nenad-saldziev/src/app/addons/mapping.ts

```diff
@@ -0,0 +1,107 @@
+import { Observable } from "rxjs";
```

Comment:
The point of the mapping is to transform the data to a better format. In this case both formats are the same, so the transformation is in place. Still, the code is ok, +5 points